### ISSUE TITLE
PI10 Streamline QA - postman collections, cron job

### ISSUE THREAD

**Issue by elnyry-sam-k** [mojaloop/project#1464]
## **Goal**:
As an Adopter of Mojaloop
I want to have _OSS postman test collections that are streamlined and well documented_
so that _I can use these collections for regression testing and ensuing overall quality of my implementation_

**Tasks**:
- [x] Address the difference (delta) between the MojaSim and Legacy postman collections so that MojaSim collection has improved coverage and includes tests added to the LegacySim collection. New story --> #1518 
 - [x] Legacy Sim to Mojasims collection  - delta New story --> #1519
       Following folders 
       -  Bug Fixes Folder (#998 , 1009, 981, 990, 1331, 1378, 849, 949, 742, 733, 956, 883, 1032)
       -  transaction request service ( health, received state, reject state, error framework, authorizations, authorizations -error framework)
       -  quoting service folder 
       -  p2p on us transfer ( 4 request)
       -  version 1.1 tests 
- [ ] ~Refactor LegacySim setup & test collections so that it can be run in parallel easily with MojaSim collections (Currently setup specific to each type needs to be run for either MojaSim or LegacySim collections to be run)~
- [x] Verify 100% pass rate with both test collections and setup collections
- [ ] Update cron jobs on OSS dev environments to run multiple collections (LegacySim, MojaSim, Bulk test collections)

**Acceptance Criteria**:
- [x] MojaSim collection is updated to include the delta with LegacySim collection
- [x] LegacySim collection is refactored so that both MojaSim and LegacySim collections can be run on the same environment without additional setup
- [x] Cron jobs are refactored to run multiple test collections

## **Pull Requests**:
- [ ] TBD

## **Follow-up**:
- N/A

**Dependencies**:
- N/A

## **Accountability**:
- Owner: TBC
- QA/Review: @elnyry-sam-k

**Comment by elnyry-sam-k** [comment]
Using TTK for cron jobs now
